templates/org.eld

```diff
@@ -25,6 +25,12 @@ org-mode
  (format-time-string "%Y-%m-%d") n ":EXPORT_HUGO_DRAFT: false" n ":END:")
 (rdmecollapse  "*** " (p "Heading") n "#+HTML: <details> <summary> " (p "sub-heading")  " </summary>" n
  (r> "link or any comments") n n "#+HTML: </details>" n)
+(python   & "#+begin_src python" n r n "#+end_src"
```

Comment:
Please rename this block. I'm using auto template,so every time I need to write python it will automatically replaced by template. I would like to avoid such cases

Reply:
Do you have any suggestions? There's py, pythn, pyt... In yasnippet, it seems like [py_](https://github.com/AndreaCrotti/yasnippet-snippets/blob/1bf034887c4048c38266842686b7f9c8384f72e7/snippets/org-mode/python#L3) is used.

Reply:
I'm fine with `py_`